packages/kyodo-contracts/package.json

```diff
@@ -15,7 +15,7 @@
     "babel-polyfill": "^6.26.0",
     "babel-preset-env": "^1.7.0",
     "babel-register": "^6.26.0",
-    "ganache-cli": "^6.1.3",
+    "ganache-cli": "6.1.8",
```

Comment:
Fixed in [ganache-cli@6.2.2](https://github.com/trufflesuite/ganache-cli/releases/tag/v6.2.2). Let me know if you have any more issues.